students/slupek_bartosz/lesson_03_functions/collatz.py

```diff
@@ -0,0 +1,15 @@
+def collatz(number):
+    if number % 2 == 0:
+        print(number // 2)
+        return number // 2
+    else:
+        print(3 * number + 1)
+        return 3 * number + 1
+
+
+try:
```

Comment:
całą tą część kodu, która już wywołuje kalkulator zamknij w warunku (to samo się dotyczy pozostałych zadań):
if name == 'main':
Dzięki czemu podczas importu funkcji z tego modułu do innego modułu/skryptu ta część modułu nie zostanie wykonana.
On się wykona tylko wtedy, gdy ten moduł/skrypt zostanie uruchomiony jak skrypt (na najwyższym poziomie).
Jeżeli uruchomisz ten skrypt zmienna name przyjmie wartość 'main', jeżeli zaimportujesz jako moduł to name ma wartość: 'nazwa_pliku'.
Pamiętaj o tym tworząc nowy moduł i możesz to tez dodać do pozostałych zadań, dzięki czemu robiąc zadania z listy 4 nie będziesz mieć problemów podczas importowaniu funkcji do testowania.
W skryptach z lekcji 0 użyty był ten mechanizm, możesz zobaczyć jak go użyto.